pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/EventService.java

```diff
@@ -27,11 +27,6 @@ public class EventService {
     protected static final int MAX_TRIES = 5;
     private static final EventDao eventDao = new EventDao();
     private static final SkippedParticipantEventDao skippedParticipantEventDao = new SkippedParticipantEventDao();
-    private static final int INITIAL_RETRY_INTERVAL = ConfigManager.getInstance().getConfig()
```

Comment:
The static initializer approach (that I had suggested earlier) turned out to be unworkable in this case, so I moved these lookups back to where they were originally, inside the method that does the retries.